app/models/request_log.rb

```diff
@@ -1,17 +1,2 @@
 class RequestLog < ApplicationRecord
```

Comment:
it might make sense if we were using create() rather than create!() as it would validate and fail rather than crash hard.
I did wonder about the whole crashing if we can't write a log record, but I'm not sure it matters.

Reply:
I've converted the non-nullable attributes into presence for safety against existing data